server/jwt_test.go

```diff
@@ -57,11 +57,14 @@ func addAccountToMemResolver(s *Server, pub, jwtclaim string) {
 	s.mu.Unlock()
 }
 
-func createClient(t *testing.T, s *Server, akp nkeys.KeyPair) (*client, *bufio.Reader, string) {
+func createClient(t *testing.T, s *Server, akp nkeys.KeyPair, optIssuerAccount string) (*client, *bufio.Reader, string) {
```

Comment:
Feels like this should be left as is and add a new createClientWithIssuers or something like that. Avoid all the ""

Reply:
Agree